testapps/testapp/build.gradle

```diff
@@ -111,6 +118,8 @@ dependencies {
     localImplementation project(':msal')
     distImplementation "com.microsoft.identity.client:msal:${msalVersion}"
     implementation "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
+    implementation "androidx.constraintlayout:constraintlayout:$rootProject.ext.constraintLayoutVersion"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$rootProject.ext.kotlinXCoroutinesVersion"
     implementation "androidx.appcompat:appcompat:$rootProject.ext.appCompatVersion"
```

Comment:
May I ask which build variant should we use? I still need to do modifications on gradle in order to compile the test app.
```
//    if (findProject(":otelexporter") != null) {
//        localImplementation(project(':otelexporter'))
//    } else {
//        localImplementation "com.microsoft.identity.client:opentelemetry.exporter:1.+"
//    }

    localImplementation "com.microsoft.identity.client:opentelemetry.exporter:1.+"
```